daprdocs/content/en/reference/api/workflow_api.md

```diff
@@ -75,6 +75,114 @@ Code | Description
 
 This API does not return any content.
 
+## Raise Event workflow request
```

Comment:
Maybe we should reword this to say "Raise Event request for a workflow" or simply "Raise Event request"